### PR TITLE
Implement syncing for IPAM release branches

### PR DESCRIPTION
**Pull request by Rozzii**
This PR also moves the custom repo update logic to a
function to provide reusable functionality.

This PR  unifies the indentation.

### CODE REVIEW

**Comment by Rozzii** [comment]
Tested in this job: https://jenkins.nordix.org/view/Airship/job/airship_nordix_dev_tools_repos/32033/console

File: scripts/update-nordix-repos-master.sh [hunk 1]
```diff
@@ -131,4 +131,19 @@ do
...
+  echo "Updating ${branch} branch in ${LOCAL_IPAM_REPO}"
+  pushd "${LOCAL_IPAM_REPO}" || exit
+  git checkout "origin/${branch}"
+  git fetch "origin" "${branch}"
+  git rebase "origin/${branch}"
+  git remote add "remote-${branch}" "${NORDIX_IPAM_REPO}"
+  git push -uf "remote-${branch}" "HEAD:refs/heads/${branch}"
+  echo "Push done to ${branch} branch in ${NORDIX_IPAM_REPO}"
+  popd || exit
+done
```
Comment (fmuyassarov) [inline scripts/update-nordix-repos-master.sh:147]: perhaps we could reuse the loop created for CAPM3 with some naming changes?

Comment (Rozzii) [reply scripts/update-nordix-repos-master.sh:147]: done.

**Comment by Rozzii** [comment]
Second test: https://jenkins.nordix.org/view/Airship/job/airship_nordix_dev_tools_repos/32325/console

**Review by fmuyassarov — APPROVED** [review]
Good job!

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Pull request merged**